### PR TITLE
refactor/527

### LINKED ISSUE #527: Update Shannon Airport (EINN)
Needs an update

### PR DESCRIPTION
**Pull request by Fechulo**
Resolves openscope/openscope#527 

### Completion Checklist
- [x] terrain file
- [x] polygonal airspace boundary
- [x] all SIDs
- [x] all STARs
- [x] information to validate traffic levels
- [x] base video map
- [x] no console errors or warnings

---

According to Wikipedia, the airport has 19,149 yearly operations. Traffic level should fall between 2.2acph and 6.6acph. Currently, there's a total of 6.6acph.

Changes:
-Updated SID's
-Added STAR's
-Added more fixes
-Updated SpawnPatterns
-Updated Traffic
-Added Terrain
-Added Video Map

---

Bring EINN up to date with the new airport specs.

### CODE REVIEW

**Comment by Fechulo** [comment]
We could do the same thing we did for Dublin, where the STAR's only used the first 4 letters of the fix name. It's not totally realistic but it's not too different and we already have a similar situation elsewhere.

**Comment by Fechulo** [comment]
Nah you're right, class C would be way more appropriate.

File: assets/airports/einn.json [hunk 1]
```diff
@@ -192,427 +362,1794 @@
...
+    "stars": {
+        "AGIN": {
+            "icao": "AGIN",
+            "name": "Aginee",
+            "suffix": {"06":"2E", "24":"2D"},
+            "entryPoints": {
+                "AGINI": [["AGINI", "A80+"]]
+            },
+            "rwy": {
+                "06": ["ULSIT", ["TERDU", "A60+"], ["ELPOM", "A30+|S220"]],
```
Comment (erikquinn) [inline assets/airports/einn.json]: speed 220 or less at ELPOM

File: assets/airports/einn.json [hunk 2]
```diff
@@ -192,427 +362,1794 @@
...
+                "24": [["OGONO", "A60+"], ["DERAG", "A30+|S220"]]
```
Comment (erikquinn) [inline assets/airports/einn.json]: speed 220 or less at DERAG

File: assets/airports/einn.json [hunk 3]
```diff
@@ -192,427 +362,1794 @@
...
+            },
+            "body": [],
```
Comment (erikquinn) [inline assets/airports/einn.json]: Can we move these `body`s between the entry and runway sections?

File: assets/airports/einn.json [hunk 4]
```diff
@@ -192,427 +362,1794 @@
...
+        "BUNO": {
+            "icao": "BUNO",
+            "name": "Boonon",
+            "suffix": {"06":"2E", "24":"2D"},
+            "entryPoints": {
+                "BUNON": [["BUNON", "A80+"]]
+            },
+            "rwy": {
+                "06": [["ELPOM", "A30+|S220"]],
+                "24": ["ENS", ["TOBRI", "A60+"], ["DERAG", "A30+|S220"]]
```
Comment (erikquinn) [inline assets/airports/einn.json]: speed 220 or less at DERAG

File: assets/airports/einn.json [hunk 5]
```diff
@@ -192,427 +362,1794 @@
...
+        },
+        "BUNO": {
+            "icao": "BUNO",
+            "name": "Boonon",
+            "suffix": {"06":"2E", "24":"2D"},
+            "entryPoints": {
+                "BUNON": [["BUNON", "A80+"]]
+            },
+            "rwy": {
+                "06": [["ELPOM", "A30+|S220"]],
```
Comment (erikquinn) [inline assets/airports/einn.json]: speed 220 or less at ELPOM

File: assets/airports/einn.json [hunk 6]
```diff
@@ -192,427 +362,1794 @@
...
+        },
+        "DIGA": {
+            "icao": "DIGA",
+            "name": "Digan",
+            "suffix": {"06":"2E", "24":"2D"},
+            "entryPoints": {
+                "DIGAN": [["DIGAN", "A80+"]]
+            },
+            "rwy": {
+                "06": ["ULSIT", ["TERDU", "A60+"], ["ELPOM", "A30+|S220"]],
```
Comment (erikquinn) [inline assets/airports/einn.json]: speed 220 or less at ELPOM

File: assets/airports/einn.json [hunk 7]
```diff
@@ -192,427 +362,1794 @@
...
+        },
+        "ERAB": {
+            "icao": "ERAB",
+            "name": "Ehrabee",
+            "suffix": {"06":"2E", "24":"2D"},
+            "entryPoints": {
+                "ERABI": [["ERABI", "A80+"]]
+            },
+            "rwy": {
+                "06": [["TERDU", "A60+"], ["ELPOM", "A30+|S220"]],
```
Comment (erikquinn) [inline assets/airports/einn.json]: speed 220 or less at ELPOM

File: assets/airports/einn.json [hunk 8]
```diff
@@ -192,427 +362,1794 @@
...
+        },
+        "KURU": {
+            "icao": "KURU",
+            "name": "Kooroom",
+            "suffix": {"06":"2E", "24":"2D"},
+            "entryPoints": {
+                "KURUM": [["KURUM", "A80+"]]
+            },
+            "rwy": {
+                "06": [["RERKA", "A60+"], ["ELPOM", "A30+|S220"]],
```
Comment (erikquinn) [inline assets/airports/einn.json]: speed 220 or less at ELPOM

File: assets/airports/einn.json [hunk 9]
```diff
@@ -192,427 +362,1794 @@
...
+        },
+        "LUPO": {
+            "icao": "LUPO",
+            "name": "Loopor",
+            "suffix": {"06":"2E", "24":"2D"},
+            "entryPoints": {
+                "LUPOR": [["LUPOR", "A80+"]]
+            },
+            "rwy": {
+                "06": [["ELPOM", "A30+|S220"]],
```
Comment (erikquinn) [inline assets/airports/einn.json]: speed 220 or less at ELPOM

File: assets/airports/einn.json [hunk 10]
```diff
@@ -192,427 +362,1794 @@
...
+        },
+        "MOMI": {
+            "icao": "MOMI",
+            "name": "Momin",
+            "suffix": {"06":"2E", "24":"2D"},
+            "entryPoints": {
+                "MOMIN": [["MOMIN", "A80+"]]
+            },
+            "rwy": {
+                "06": [["TERDU", "A60+"], ["ELPOM", "A30+|S220"]],
```
Comment (erikquinn) [inline assets/airports/einn.json]: speed 220 or less at ELPOM

File: assets/airports/einn.json [hunk 11]
```diff
@@ -192,427 +362,1794 @@
...
+        },
+        "RIKU": {
+            "icao": "RIKU",
+            "name": "Ricool",
+            "suffix": {"06":"2E", "24":"2D"},
+            "entryPoints": {
+                "RIKUL": [["RIKUL", "A80+"]]
+            },
+            "rwy": {
+                "06": ["ULSIT", ["TERDU", "A60+"], ["ELPOM", "A30+|S220"]],
```
Comment (erikquinn) [inline assets/airports/einn.json]: speed 220 or less at ELPOM

File: assets/airports/einn.json [hunk 12]
```diff
@@ -192,427 +362,1794 @@
...
+        },
+        "TIPU": {
+            "icao": "TIPU",
+            "name": "Tipoor",
+            "suffix": {"06":"2E", "24":"2D"},
+            "entryPoints": {
+                "TIPUR": [["TIPUR", "A80+"]]
+            },
+            "rwy": {
+                "06": [["RERKA", "A60+"], ["ELPOM", "A30+|S220"]],
```
Comment (erikquinn) [inline assets/airports/einn.json]: speed 220 or less at ELPOM

File: assets/airports/einn.json [hunk 13]
```diff
@@ -192,427 +362,1794 @@
...
+        },
+        "UNBE": {
+            "icao": "UNBE",
+            "name": "Unbeg",
+            "suffix": {"06":"2E", "24":"2D"},
+            "entryPoints": {
+                "UNBEG": [["UNBEG", "A80+"]]
+            },
+            "rwy": {
+                "06": [["ELPOM", "A30+|S220"]],
```
Comment (erikquinn) [inline assets/airports/einn.json]: speed 220 or less at ELPOM

File: assets/airports/einn.json [hunk 14]
```diff
@@ -192,427 +362,1794 @@
...
+        "DIGA": {
+            "icao": "DIGA",
+            "name": "Digan",
+            "suffix": {"06":"2E", "24":"2D"},
+            "entryPoints": {
+                "DIGAN": [["DIGAN", "A80+"]]
+            },
+            "rwy": {
+                "06": ["ULSIT", ["TERDU", "A60+"], ["ELPOM", "A30+|S220"]],
+                "24": [["OGONO", "A60+"], ["DERAG", "A30+|S220"]]
```
Comment (erikquinn) [inline assets/airports/einn.json]: speed 220 or less at DERAG

File: assets/airports/einn.json [hunk 15]
```diff
@@ -192,427 +362,1794 @@
...
+        "ERAB": {
+            "icao": "ERAB",
+            "name": "Ehrabee",
+            "suffix": {"06":"2E", "24":"2D"},
+            "entryPoints": {
+                "ERABI": [["ERABI", "A80+"]]
+            },
+            "rwy": {
+                "06": [["TERDU", "A60+"], ["ELPOM", "A30+|S220"]],
+                "24": ["ENS", ["TOBRI", "A60+"], ["DERAG", "A30+|S220"]]
```
Comment (erikquinn) [inline assets/airports/einn.json]: speed 220 or less at DERAG

File: assets/airports/einn.json [hunk 16]
```diff
@@ -192,427 +362,1794 @@
...
+        "KURU": {
+            "icao": "KURU",
+            "name": "Kooroom",
+            "suffix": {"06":"2E", "24":"2D"},
+            "entryPoints": {
+                "KURUM": [["KURUM", "A80+"]]
+            },
+            "rwy": {
+                "06": [["RERKA", "A60+"], ["ELPOM", "A30+|S220"]],
+                "24": [["GILOG", "A60+"], ["DERAG", "A30+|S220"]]
```
Comment (erikquinn) [inline assets/airports/einn.json]: speed 220 or less at DERAG

File: assets/airports/einn.json [hunk 17]
```diff
@@ -192,427 +362,1794 @@
...
+        "LUPO": {
+            "icao": "LUPO",
+            "name": "Loopor",
+            "suffix": {"06":"2E", "24":"2D"},
+            "entryPoints": {
+                "LUPOR": [["LUPOR", "A80+"]]
+            },
+            "rwy": {
+                "06": [["ELPOM", "A30+|S220"]],
+                "24": ["ENS", ["TOBRI", "A60+"], ["DERAG", "A30+|S220"]]
```
Comment (erikquinn) [inline assets/airports/einn.json]: speed 220 or less at DERAG

File: assets/airports/einn.json [hunk 18]
```diff
@@ -192,427 +362,1794 @@
...
+        "MOMI": {
+            "icao": "MOMI",
+            "name": "Momin",
+            "suffix": {"06":"2E", "24":"2D"},
+            "entryPoints": {
+                "MOMIN": [["MOMIN", "A80+"]]
+            },
+            "rwy": {
+                "06": [["TERDU", "A60+"], ["ELPOM", "A30+|S220"]],
+                "24": ["ENS", ["TOBRI", "A60+"], ["DERAG", "A30+|S220"]]
```
Comment (erikquinn) [inline assets/airports/einn.json]: speed 220 or less at DERAG

File: assets/airports/einn.json [hunk 19]
```diff
@@ -192,427 +362,1794 @@
...
+        "RIKU": {
+            "icao": "RIKU",
+            "name": "Ricool",
+            "suffix": {"06":"2E", "24":"2D"},
+            "entryPoints": {
+                "RIKUL": [["RIKUL", "A80+"]]
+            },
+            "rwy": {
+                "06": ["ULSIT", ["TERDU", "A60+"], ["ELPOM", "A30+|S220"]],
+                "24": [["OGONO", "A60+"], ["DERAG", "A30+|S220"]]
```
Comment (erikquinn) [inline assets/airports/einn.json]: speed 220 or less at DERAG

File: assets/airports/einn.json [hunk 20]
```diff
@@ -192,427 +362,1794 @@
...
+        "TIPU": {
+            "icao": "TIPU",
+            "name": "Tipoor",
+            "suffix": {"06":"2E", "24":"2D"},
+            "entryPoints": {
+                "TIPUR": [["TIPUR", "A80+"]]
+            },
+            "rwy": {
+                "06": [["RERKA", "A60+"], ["ELPOM", "A30+|S220"]],
+                "24": [["GILOG", "A60+"], ["DERAG", "A30+|S220"]]
```
Comment (erikquinn) [inline assets/airports/einn.json]: speed 220 or less at DERAG

File: assets/airports/einn.json [hunk 21]
```diff
@@ -192,427 +362,1794 @@
...
+        "UNBE": {
+            "icao": "UNBE",
+            "name": "Unbeg",
+            "suffix": {"06":"2E", "24":"2D"},
+            "entryPoints": {
+                "UNBEG": [["UNBEG", "A80+"]]
+            },
+            "rwy": {
+                "06": [["ELPOM", "A30+|S220"]],
+                "24": ["ENS", ["TOBRI", "A60+"], ["DERAG", "A30+|S220"]]
```
Comment (erikquinn) [inline assets/airports/einn.json]: speed 220 or less at DERAG

**Comment by Fechulo** [comment]
@erikquinn all done! In this case, the holds shown on the charts are not mandatory, it's basically a way of saying "**If** ATC tells you to hold, this is the way to do it"

**Pull request merged**